aws/subscriber.go

```diff
@@ -83,6 +83,13 @@ func (s *awsSubscriber) ExtendAckDeadline(ctx context.Context, messageID string,
 	panic("not implemented")
 }
 
+func (s *awsSubscriber) DeleteSubscription() error {
+	_, error := s.sqs.DeleteQueue(&sqs.DeleteQueueInput{
```

Comment:
go naming conventions: call this `err`, not `error`. I'm surprised this would even build...